### DISCUSSION TITLE
Add DEB / RPM packages via packager.io

### DISCUSSION THREAD

**Discussion by monotek** [postalserver/postal#1466] [Feature suggestions]
https://packager.io/ provides free DEB / RPM package builds for open source projects on Ubuntu, Debian, Centos and more. 
Ruby is supported so would be nice to have automated package builds.

https://packager.io/documentation/

**Comment by marianord** [comment]
Has this been thought up? I'd love to install Postal on Debian and right now you're only officially supporting an install on Ubuntu 16.04 which is about to end support, and is already superseded by 18.04 and 20.04

**Comment by willpower232** [comment]
@MarianoRD FWIW the dependencies have been updated for more recent Ruby versions so if you wanted to set it up with some strong hints from the installation script then hopefully you wouldn't run into many troubles but yes it would be nice to have more documented support for newer operating systems
